p2panda-rs/src/storage_provider/traits/entry_store.rs

```diff
@@ -65,26 +173,41 @@ pub trait EntryStore<StorageEntry: AsStorageEntry> {
 
         entry_skiplink_hash
     }
+
+    /// Get all entries which make up the certificate pool for the given entry.
+    ///
+    /// Returns a result containing vector of entries wrapped in an option. If no entry
+    /// could be found at this author - log - seq number location then an error is
+    /// returned.
+    async fn get_all_lipmaa_entries_for_entry(
```

Comment:
We've been using the "skiplink" name in the API so far and I think it would be good to not add another term here.